backend/src/Edna/Library/Web/Schema.hs

```diff
@@ -0,0 +1,11 @@
+module Edna.Library.Web.Schema
```

Comment:
I don't see a benefit in this module. It's small and has an orphan instance. This instance can be moved to `Edna.Library.Web.Types` harmlessly. I could understand this module if we had 100 library web types, but we are going to have about 10-20 and each instance is going to be rather small and trivial, so I think it's better to keep them in the same module as type definitions. And currently there is inconsistency because JSON instances are defined in `Types` while swagger instances are here.